perun-core/src/main/java/cz/metacentrum/perun/core/impl/GroupsManagerImpl.java

```diff
@@ -258,6 +259,31 @@ public Group updateGroupName(PerunSession sess, Group group) throws InternalErro
 		return dbGroup;
 	}
 
+	public Group updateParentGroupId(PerunSession sess, Group group) throws InternalErrorException {
+		Utils.notNull(group.getName(), "group.getName()");
```

Comment:
Is this check necessary ? Technically, name should always exists, but we do not check other params and you do not work with group name in this method.